src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown

```diff
@@ -151,3 +151,34 @@ SAP Concur services will send the following HTTP headers in every request. Pleas
 `Accept`|`string`|-| SAP Concur will always set the `Accept` header to `application/json`.|
 `concur-correlationid`|`string`|-| In order to assist with troubleshooting, SAP Concur provides a unique `correlationId` in the request header. This unique code can be used during troubleshooting as it identifies the API call in the log files.|
 `Accept-Language`|`string`|-| Value of this header will follow language tag of the format LanguageCode-CountryCode, e.g. `en-US`, `fr-CA`, `de-CH` etc.|
+
+## Confirmation Codes <a name="confirmation-codes"></a>
+
+Please note current definitions of `ConfirmationCodeType` differ from what was used in [Hotel Service v2](./hotel-service-2/Introduction.html) (HSv2).
+* `RESERVATION`: Maps to OTA Unique ID Type `14` and has no equivalent for HSv2. This will be the record locator of supplier PNR and will be provided in all subsequent read/modify/cancel request on a booking. This is optional and will be mostly pass through for SAP Concur systems.
+* `SUPPLIER_CONFIRMATION`: **(Required)** Maps to OTA Unique ID Type `40` and represents supplier's confirmation number. This also be provided in all subsequent read/modify/cancel request on a booking, and it is also used in the passive segment created in the GDS. (Equivalent to `14` in HSv2).
+* `HOTEL_CONFIRMATION`: Maps to OTA Unique ID Type `10`. Used by travelers if they want to change their reservation outside of SAP Concur Online Booking Tool. If the property confirmation code is not available, the supplier may send the aggregator confirmation code in this field. (Equivalent to `1000` in HSv2). This field will appear on the itinerary page together with `SUPPLIER_CONFIRMATION` confirmation number.
+* `CANCELLATION`: **(Required)** Maps to OTA Unique ID Type `15` and is same as HSv2 and must be provided as a part of `CancelDetails` response object.
+* `CONCUR_GDS_REFERENCE`: This is custom code for **future** use (Type `14`) which will be sent by SAP Concur in read/modify requests and will provide Concur Booking Record Locator for passives.
+
+## Error Handling <a name="error-handling"></a>
+
+All HSv4 endpoints use consistent error handling. API responses can have one of the following HTTP response code:
+
+|HTTP Code|Description|
+|---|---|
+|200|Operation successful without any errors.|
+|400|Invalid client request. Request shouldn’t be retried without changing it. Response should follow [Error](./v4.endpoints.md#schemaerror) schema and include OTA Code providing accurate reason for request being invalid.| 
+|401|Request unauthorized. Credentials used for the request are not valid.|
+|500|System error while processing the request. Request can be retried as is at a later time. Response should follow [Error](./v4.endpoints.md#schemaerror) schema. OTA Code is optional for such errors but we expect useful message for better troubleshooting.| 
+
```

Comment:
@kunalshah01  Do you see the value in adding following error codes as well to the above list?
404 -Not Found, 503 - Indicate Outage at the vendor side

Reply:
No, expectation is to use OTA codes for all validation/application errors and 500 for all system errors (including outage).